config/jobs/kubernetes/sig-testing/bazel-build-test.yaml

```diff
@@ -303,11 +303,13 @@ periodics:
       - --
       - -//build/...
       - -//vendor/...
+
 - name: periodic-kubernetes-bazel-test-canary
+  cluster: k8s-infra-prow-build
   annotations:
     testgrid-dashboards: sig-testing-canaries
     testgrid-tab-name: bazel-test
-    description: run kubernetes-bazel-test with latest image
+    description: run kubernetes-bazel-test without RBE on k8s-infra-prow-build
```

Comment:
is "without RBE" necessary? only test-infra is using it currently, nominally

Reply:
yes, there are kubernetes/kubernetes jobs that use RBE either via `--config=ci` or `--config=remote` (ref: https://github.com/kubernetes/kubernetes/blob/master/build/root/.bazelrc#L52-L87)

e.g.
- periodic-bazel-test-master above
- periodic-bazel-build-based